pom.xml

```diff
@@ -45,9 +47,58 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+
+			<!-- HANA -->
+			<dependency>
+				<groupId>com.sap.cloud.db.jdbc</groupId>
+				<artifactId>ngdbc</artifactId>
+				<version>${ngdbc.version}</version>
+			</dependency>
 		</dependencies>
 	</dependencyManagement>
 
+	<profiles>
```

Comment:
I wouldn't have this profile section here, this just introduces more complexity to the build (and if at all should be placed into the `srv/pom.xml`). The only advantage we gain from this is that we could potentially exclude the sqlite JDBC driver from the productive app, but this is not so important, as it is anyways inactive. And for the sake of simplicity in that example I wouldn't introduce that complexity here.

Reply:
Yes this simplified it quite a bit!